core/src/scene/styleParam.cpp

```diff
@@ -180,6 +184,17 @@ StyleParam::Value StyleParam::parseString(StyleParamKey key, const std::string&
         }
         return vec.value;
     }
+    case StyleParamKey::text_buffer:
```

Comment:
Can `buffer` param take stops as a value? I could not find it in the documentation, but probably good to confirm from JS implementation.